package/readme.txt

```diff
@@ -47,6 +47,9 @@ Localization:
 Bug fixes:
 - The Start Menu group the application is placed in is now named 'muCommander' instead of 'Unknown' on Windows.
 - The application sometimes cannot start after being killed.
+- Fixed unnecessary stack-traces in the logs while starting or running the appliaction.
```

Comment:
I'd say it's an improvement rather than a fix, wdyt?

Reply:
ok, moving to improvements section.